### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by sryu1** [SurferTim/CitationX#1]
I just came here from a link from the forum and it was last updated 6 hrs ago although the forum post was 2 years ago... 😕 
![image](https://user-images.githubusercontent.com/95025816/204401364-4c2d34f2-e1d8-4ca1-baca-5847ee62974f.png)

**Comment by SurferTim** [comment]
Some videos are a couple years old. I have continued to modify the aircraft, along with the Twin Otter. This update added compositor lights and a new autopilot feature (heading to intercept VOR radial or localizer.)

**Comment by sryu1** [comment]
Is this based of the PAF version of the Cessna Citation X?

**Comment by SurferTim** [comment]
I downloaded the basic aircraft years ago from here:
http://mirrors.ibiblio.org/flightgear/ftp/Aircraft/
The Patten hangar model was released after that. I could not get the Patten hangar Citation to work. It would crash FG about 15 minutes into a flight. That ended up being a FG problem, now resolved on my system.

The reason I had to delete and reload the Citation yesterday was my .git file had been corrupted.

If you try the original CitationX from the link above, the MFD won't work. It is now obsolete, and has been replaced on my version with a canvas MFD.
